### ISSUE TITLE
Fix notebooks not executable in CI

### ISSUE THREAD

**Issue by vuillaut** [cta-observatory/cta-lstchain#1295]
I propose that all authors go through their notebook and see:
- if it can run - and if not update it 
- if it can run with the test data available in the CI rather than a custom path
- delete it if completely obsolete

Here is a list of all notebooks and their original author.
Please open a PR to fix your notebooks and check the boxes here.

- [ ] DL1b_to_DL2.ipynb: @SeiyaNozaki
- [ ] DL2_to_DL3.ipynb: @chaimain
- [ ] MC_DL2_to_IRF.ipynb: @chaimain
- [ ] apply_calibration.ipynb: @FrancaCassol
- [ ] apply_calibration_with_gain_selection.ipynb: @FrancaCassol
- [ ] calculate_calibration_coefficients.ipynb: @FrancaCassol
- [ ] calculate_pedestals.ipynb: @FrancaCassol
- [ ] calculate_sensitivity_eventio.ipynb: @rlopezcoto
- [ ] calculate_sensitivity_from_Crab.ipynb: @moralejo
- [ ] calculate_sensitivity_random_data.ipynb: @rlopezcoto
- [ ] cleaning_dl1_with_pedestal.ipynb: @pawel21
- [ ] data_quality.ipynb: @moralejo
- [ ] disp.ipynb: @vuillaut
- [ ] example_container.ipynb: @vuillaut
- [ ] explore_DL2.ipynb: @moralejo
- [ ] gain_estimation.ipynb:  @Yukiho Kobayashi
- [ ] low_level_corrections.ipynb: @pawel21
- [ ] lst_sensitivity_pyirf.ipynb: @misabelber
- [ ] lstchain.ipynb: @misabelber
- [ ] mc_energy_distr.ipynb: @vuillaut
- [ ] plot_mc_muon_parameters.ipynb: @rlopezcoto
- [ ] plot_muon_data_mc_comparisons.ipynb: @rlopezcoto
- [ ] post_DL3_analysis.ipynb: @chaimain
- [ ] read_LST1_data.ipynb: @FrancaCassol
- [ ] read_and_plot_LST_Data.ipynb: @rlopezcoto
- [ ] rf-performance.ipynb: @rlopezcoto
- [ ] significance_calculation.ipynb: @rlopezcoto
- [ ] source_dependent_analysis.ipynb: @SeiyaNozaki
- [ ] step_by_step_dl0_to_dl1.ipynb: @rlopezcoto
- [ ] tests_on_impact_parameter.ipynb: @rlopezcoto
- [ ] time_calibration.ipynb: @pawel21

**Comment by morcuended** [comment]
thanks for the effort @vuillaut

**Comment by moralejo** [comment]
Both calculate_sensitivity_from_Crab and data_quality work, but need as input very long samples to produce meaningful output. I think it is not feasible to have such samples in the standard test data. The first of the two also takes *very* long to finish. What can one do in such cases?
As for explore_DL2, it could do with O(1h) of Crab, but I do not think we have such a DL2 sample in the test data, right?

**Comment by morcuended** [comment]
> As for explore_DL2, it could do with O(1h) of Crab, but I do not think we have such a DL2 sample in the test data, right?

No, we've just got a few hundred raw-data events. However, we could think of actually including these 1h-data DL2 files trimming the parameters not needed for the theta2 calculation from DL2 files.

For the other two notebooks, it's hard to say. Actually, the data quality notebook could be transformed into a script, that could be more easily tested. Although the notebook give you the interactive part, that is also helpful.
Concerning the sensitivity one, I agree it'd be hard to include it in the CI and use test data. But it'd be great if somehow this could be checked automatically for every (major) lstchain release and compare it with sensitivity calculated with previous version. But it goes beyond the scope of this PR.

I think it is enough for these three notebooks that they can run and are up to date (in this case checked manually)
